examples/flutter_todos/.metadata

```diff
@@ -4,7 +4,7 @@
 # This file should be version controlled and should not be manually edited.
```

Comment:
Should this file be committed? What purpose does it serve?

Reply:
It's a file used by the Flutter framework to keep track of versioning information. It should be version controlled, i.e., committed into the codebase 😄